<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FIDMAA</string>
  </property>
  <widget class="QClickableLabel" name="chartLabel">
   <property name="geometry">
    <rect>
     <x>786</x>
     <y>12</y>
     <width>255</width>
     <height>640</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>255</width>
     <height>640</height>
    </size>
   </property>
   <property name="text">
    <string>Chart</string>
   </property>
  </widget>
  <widget class="QClickableLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>480</width>
     <height>640</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>640</height>
    </size>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>263</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="loadJPEGButton">
        <property name="text">
         <string>&amp;Load image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open3DViewButton">
        <property name="text">
         <string>&amp;Open 3D view</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="showZoomWindowButton">
      <property name="text">
       <string>&amp;Open zoom window</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>x:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xValue">
        <property name="maximum">
         <number>480</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>y:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="yValue">
        <property name="maximum">
         <number>640</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>angle (skew):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="angleValue">
        <property name="suffix">
         <string> deg</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>45</number>
        </property>
        <property name="maximum">
         <number>135</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="angleSlider">
      <property name="minimum">
       <number>45</number>
      </property>
      <property name="maximum">
       <number>135</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="dataOutputEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>380</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionShow_chart">
   <property name="text">
    <string>Show chart</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QClickableLabel</class>
   <extends>QLabel</extends>
   <header>qclickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>angleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
